gst/nnstreamer/nnstreamer_subplugin.c

```diff
@@ -91,19 +133,11 @@ get_subplugin (subpluginType type, const char *name)
     guint i;
     guint ret = nnsconf_get_subplugin_info (conf_type, &info);
 
+    G_UNLOCK (splock);
     for (i = 0; i < ret; i++) {
-      const gchar *fullpath = info.paths[i];
-
-      dlerror ();
-      G_UNLOCK (splock);
-      handle = dlopen (fullpath, RTLD_NOW);
-      /* If this is a correct subplugin, it will register itself */
-      G_LOCK (splock);
-      if (NULL == handle) {
-        g_critical ("Cannot dlopen %s with error %s.", fullpath, dlerror ());
-        continue;
-      }
+      _search_subplugin (type, info.names[i], info.paths[i]);
```

Comment:
Now no need to check whether it's successful or not?

Reply:
currently the result is ignored here for the full-scan and registration. actually result will be checked in line 155.